src/Nancy.Serialization.JsonNet/Helpers.cs

```diff
@@ -2,8 +2,39 @@
 {
     using System;
 
+    using Nancy.Json;
+
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Serialization;
+
     internal static class Helpers
     {
+        public static JsonSerializer DefaultSerializer()

```

Comment:
`JsonSerializer.CreateDefault ` is json.net so since we call this in more than one spot I figured it was best to move it to the helper class. This method can be cleaned up a bit though after looking over it again.